src/containers/AutoProvision/index.js

```diff
@@ -92,7 +92,7 @@ const AutoProvision = ({
   };
 
   const onSubmit = () => {
-    const formattedData = { ...data.details };
+    const formattedData = JSON.parse(JSON.stringify(data.details));
```

Comment:
this isn't the right solution. The issue is you're trying to edit a prop because you're not cloning the object properly. Look at the Profiles code for formattedData